src/statistical_analysis.py

```diff
@@ -0,0 +1,229 @@
+import json
```

Comment:
Since we have az `Analysis` class, it is not clear what is the scope of this class. I am quite sure, that we need (at least) two classes, but I do not see a clear separation between them.

Reply:
Analysis class was renamed to GraphAnalysis and any function that needs a graph as an input is now in this class. Functions that create graphs (yearly) or calculate statistics simply by reading files are in StatisticalAnalysis.

Reply:
Update: I needed an AnalysisHandler class too